### PR TITLE
Added Microsoft, Daisy Disk, and Plex icons

### PR DESCRIPTION
**Pull request by austincondiff**
This PR introduces the following icons.

|Name | Original | Proposed |
|:-------------------------|:-------------------------:|:-------------------------:|
| Daisy Disk | original Daisy Disk icon | Big Sur Daisy Disk icon |
| Microsoft Excel Alternate | original Microsoft Excel Alternate icon | Big Sur Microsoft Excel Alternate icon |
| Microsoft OneDrive Alternate | original Microsoft OneDrive Alternate icon | Big Sur Microsoft OneDrive Alternate icon |
| Microsoft OneNote Alternate | original Microsoft OneNote Alternate icon | Big Sur Microsoft OneNote Alternate icon |
| Microsoft Outlook Alternate | original Microsoft Outlook Alternate icon | Big Sur Microsoft Outlook Alternate icon |
| Microsoft PowerPoint Alternate | original Microsoft PowerPoint Alternate icon | Big Sur Microsoft PowerPoint Alternate icon |
| Microsoft Teams Alternate | original Microsoft Teams Alternate icon | Big Sur Microsoft Teams Alternate icon |
| Microsoft To Do Alternate | original Microsoft To Do Alternate icon | Big Sur Microsoft To Do Alternate icon |
| Microsoft Visual Studio Code Alternate | original icrosoft VS Code Alternate icon | Big Sur icrosoft VS Code Alternate icon |
| Microsoft Word Alternate | original Microsoft Word Alternate icon | Big Sur Microsoft Word Alternate icon |
| Plex | original Plex icon | Big Sur Plex icon |

In addition, I've added a `.gitignore` file which prevents `.DS_Store` files from cluttering up the repo.

### CODE REVIEW

**Comment by 77Z** [comment]
👍 I like em

File: icns.txt [hunk 1]
```diff
@@ -186,7 +197,7 @@ Sublime_Text,
 Swift_Playgrounds,
 Swift_Playgrounds_Alt,
 Swift_Playgrounds_Alt2,
-table-plus,
+Table Plus,
```
Comment (dimitarnestorov) [inline icns.txt:200]: Shouldn't this be with an underscore?

File: icns.txt [hunk 2]
```diff
@@ -141,7 +151,7 @@ NordPass,
 NordVPN,
 NordVPN_alt2,
 Notability,
-intellij,
```
Comment (elrumo) [inline icns.txt:144]: The app names should have the exactly same name as the icon names, as this list is being used by the website to fetch all the icons. Hence why "intellij" is in lower case, to be the same as the icon files.

Comment (austincondiff) [reply icns.txt:144]: I meant to change that file name, thanks for catching that!

File: icns.txt (hunk 1 above)
Comment (austincondiff) [reply icns.txt:200]: Good catch

**Pull request merged**